dynamic-layers/virtualization/recipes-azure-iot/iotedge/aziot-edged/aziot-edged/fix_warnings_aziot-cert-client-async.patch

```diff
@@ -0,0 +1,71 @@
+diff --git a/aziotctl/aziotctl-common/src/host_info.rs b/aziotctl/aziotctl-common/src/host_info.rs
+index 862485c..ba2c924 100644
+--- a/aziotctl/aziotctl-common/src/host_info.rs
++++ b/aziotctl/aziotctl-common/src/host_info.rs
+@@ -11,8 +11,6 @@ use serde_with::skip_serializing_none;
+ const BITNESS: usize = 32;
+ #[cfg(target_pointer_width = "64")]
+ const BITNESS: usize = 64;
+-#[cfg(target_pointer_width = "128")]
+-const BITNESS: usize = 128;
```

Comment:
why don't we solve it via `unexpected_cfgs`?

Reply:
yes could be solved via unexpected_cfgs, done.